src/RevProc.cc

```diff
@@ -46,6 +47,13 @@ RevProc::RevProc( unsigned Id,
     coProc = NULL;
   }
 
+  // Create the Hart Objects
+  for( size_t i=0; i<numHarts; i++ ){
+    Harts.emplace_back(std::make_unique<RevHart>(i));
+    HART_CTS.emplace_back(false);
+    HART_CTE.emplace_back(false);
+  }
```

Comment:
This explicit initialization of `HART_CTS` and `HART_CTE` are not necessary if the constructor above has:
```c++
HART_CTS(NumHarts), HART_CTE(NumHarts)
```
A vector constructed with a size argument value-initializes (zeros or default-constructs) its members. For a vector of `bool`, this means initializing a vector of `false`.

Right now `HART_CTS` and `HART_CTE` are default-initialized to size-0 vectors since they are not listed in the constructor, and then they are appended on. But you can initialize them to `numHarts` vectors of `false` by initializing them with their `numHarts` size.